pacquet/npm/napi/scripts/generate-packages.mjs

```
@@ -0,0 +1,94 @@
+    main: NATIVE_ADDON_FILE,
+    files: [NATIVE_ADDON_FILE],
+    repository: { type: 'git', url: 'https://github.com/pnpm/pnpm' },
+  }
+  if (target.libc) {
+    manifestData.libc = [target.libc]
+  }
+  fs.writeFileSync(resolve(packageRoot, 'package.json'), `${JSON.stringify(manifestData, null, 2)}\n`)
+
+  const source = resolve(REPO_ROOT, `${ARTIFACT_BASE}.${target.codeTarget}.node`)
+  if (!fs.existsSync(source)) {
+    console.warn(`WARN: missing prebuilt artifact ${source}; skipping ${packageName}`)
```

ESLint

[error] 'console' is not defined.

(no-undef)